aiogram/fsm/storage/redis.py

```diff
@@ -127,6 +127,18 @@ async def get_data(
             value = value.decode("utf-8")
         return cast(Dict[str, Any], self.json_loads(value))
 
+    @overload
+    async def get_value(self, storage_key: StorageKey, dict_key: str) -> Optional[Any]: ...
+
+    @overload
+    async def get_value(self, storage_key: StorageKey, dict_key: str, default: Any) -> Any: ...
+
+    async def get_value(
+        self, storage_key: StorageKey, dict_key: str, default: Optional[Any] = None
```

Comment:
You don't need to override this method in the Redis Storage because the base implementation is the same as you have added.